client/upe-onboarding-wizard/upe-preview-methods-selector/enable-upe-preview-task.js

```diff
@@ -25,7 +18,7 @@ const EnableUpePreviewTask = () => {
 		setIsUpeEnabled( true ).then( () => {
 			setCompleted( true, 'add-payment-methods' );
 		} );
-	}, [ setCompleted ] );
+	}, [ setCompleted, setIsUpeEnabled ] );
```

Comment:
In some cases these types of changes can cause the component to update often enough that the site hangs; did we test to make sure that's not the case here?

Reply:
Yeah, thanks for checking! This was a miss on my part :)

Reply:
A miss as in "we don't need to change this back" or a miss as in "this caused the site to crash"? I'm not quite sure whether there's an action needed here or not 😅

Reply:
A miss on my part when I was reviewing these files, sorry for the confusion :D no action needed, this is still working as expected 👍